### PR TITLE
Unify storing of CRDs

### PR DESCRIPTION
**Pull request by baijum**
Fixes https://jira.coreos.com/browse/ODC-496

### CODE REVIEW

File: make/test.mk [hunk 1]
```diff
@@ -80,6 +80,7 @@ ifeq ($(OPENSHIFT_VERSION),3)
 endif
 	$(eval package_yaml := ./manifests/devconsole/devconsole.package.yaml)
 	$(eval devconsole_version := $(shell cat $(package_yaml) | grep "currentCSV"| cut -d "." -f2- | cut -d "v" -f2 | tr -d '[:space:]'))
+	$(Q)cp ./deploy/crds/*.yaml ./manifests/devconsole/$(devconsole_version)/
```
Comment (Avni-Sharma) [inline make/test.mk]: do we still need the test/sanitychecks/crd_test.go after this?

Comment (baijum) [reply make/test.mk]: Good catch!
The test is failing now.
I will remove it.

Comment (baijum) [reply make/test.mk]: Done. 07f351b3cd03fade0525ad7320d86a6cca56691a

**Comment by baijum** [comment]
Looks like the test is failing for a different reason:
```
ERROR:operatorcourier.validate:custom resource definition components.devconsole.openshift.io referenced in csv not defined in root list of crds
ERROR:operatorcourier.validate:custom resource definition gitsources.devconsole.openshift.io referenced in csv not defined in root list of crds
ERROR:operatorcourier.validate:custom resource definition gitsourceanalyses.devconsole.openshift.io referenced in csv not defined in root list of crds
ERROR:operatorcourier.api:Bundle failed validation.
Resulting bundle is invalid, input yaml is improperly defined.
```

I don't understand the error message!

**Comment by baijum** [comment]
> Need to make sure all tests passes

I am not able to reproduce the test failure locally.

**Comment by akashshinde** [comment]
@baijum do we same version of `operator-sdk` installed in local ?

**Comment by akashshinde** [comment]
@baijum look like you would need to add some annotations to metadata in csv, please check https://github.com/operator-framework/operator-courier/blob/6e54f0ba6ccc9416397938e6724091878449c7c6/operatorcourier/validate.py#L268

**Comment by baijum** [comment]
> @baijum looks like you would need to add some annotations to metadata in csv, please check https://github.com/operator-framework/operator-courier/blob/6e54f0ba6ccc9416397938e6724091878449c7c6/operatorcourier/validate.py#L268

Thanks, @akashshinde! This explains the warnings, but not the errors.

**Comment by baijum** [comment]
The copying of CRDs was missing. Fixed it here: d9564eadddd4f89651f811ce38c8a49fec307d4e

**Pull request merged**